src/main/python/rlbot/botmanager/bot_manager.py

```diff
@@ -147,19 +147,23 @@ def run(self):
                 last_call_real_time = datetime.now()
 
                 # Reload the Agent if it has been modified.
-                new_module_modification_time = os.stat(agent_class_file).st_mtime
-                if new_module_modification_time != last_module_modification_time:
-                    try:
-                        last_module_modification_time = new_module_modification_time
+                try:
+                    new_module_modification_time = os.stat(agent_class_file).st_mtime
```

Comment:
could do an if check for existence instead of wrapping this all in a try catch?
I know that is not pythonic but it feels cleaner

Reply:
I'd rather do it this way partly because we don't have to worry about a race condition where the existence check check passes, then the file gets locked or moved, and the os.stat fails.